src/main/java/net/snowflake/ingest/streaming/internal/ArrowFlusher.java

```diff
@@ -0,0 +1,127 @@
+package net.snowflake.ingest.streaming.internal;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.channels.Channels;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import net.snowflake.ingest.utils.*;
```

Comment:
minor: * import. 
Might change your intellij settings. I am not sure if running `format.sh ` will fix this. (Formatter is google-java-formatter